service/view_building_coordinator.cc

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright (C) 2025-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+ */
+
+#include <exception>
+#include <iterator>
+#include <ranges>
+#include <seastar/core/coroutine.hh>
+#include <seastar/coroutine/maybe_yield.hh>
+#include <seastar/core/abort_source.hh>
+
+#include "cql3/query_processor.hh"
+#include "db/schema_tables.hh"
+#include "db/system_keyspace.hh"
+#include "mutation/canonical_mutation.hh"
+#include "schema/schema_fwd.hh"
+#include "service/raft/group0_state_machine.hh"
+#include "service/raft/raft_group0.hh"
+#include "service/raft/raft_group0_client.hh"
+#include "service/topology_coordinator.hh"
+#include "utils/assert.hh"
+#include "utils/log.hh"
+#include "service/migration_manager.hh"
+#include "replica/database.hh"
+#include "view_info.hh"
+
+#include "service/view_building_coordinator.hh"
+
+static logging::logger vbc_logger("view_building_coordinator");
+
+namespace service {
+
+view_building_coordinator::view_building_coordinator(abort_source& as, replica::database& db, raft::server& raft, raft_group0& group0, db::system_keyspace& sys_ks, const topology_state_machine& topo_sm, const raft::term_t term) 
+    : _db(db)
+    , _raft(raft)
+    , _group0(group0)
+    , _sys_ks(sys_ks)
+    , _topo_sm(topo_sm)
+    , _term(term)
+    , _as(as) 
+{}
+
+future<group0_guard> view_building_coordinator::start_operation() {
+    auto guard = co_await _group0.client().start_operation(_as);
+    if (_term != _raft.get_current_term()) {
+        throw term_changed_error{};
+    }
+    co_return std::move(guard);
+}
```

Comment:
This is topology_coordinator::start_operation() but with less logging. If this is the only duplication it may be OK.